src/storage/txn/scheduler.rs

```diff
@@ -256,7 +256,7 @@ impl<E: Engine> Scheduler<E> {
     ) -> Self {
         // Add 2 logs records how long is need to initialize TASKS_SLOTS_NUM * 2048000 `Mutex`es.
         // In a 3.5G Hz machine it needs 1.3s, which is a notable duration during start-up.
-        info!("Scheduler::new is called to initialize the transaction scheduler");
+        debug!("Scheduler::new is called to initialize the transaction scheduler");
```

Comment:
How about logging only when it takes too long?

Reply:
How long is too long? I am also curious how we can make this more relevant to the reader.

Reply:
Ping

Reply:
We have a slow log macros, maybe it can be used directly

Reply:
PTAL at https://github.com/tikv/tikv/pull/4768/commits/fffc5e76d11369f75f80adacd639a7a6d6e9e4c4